actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -320,7 +320,7 @@ def word_wrap(text, line_width: 80, break_sequence: "\n")
       #   simple_format("<a target=\"_blank\" href=\"http://example.com\">Continue</a>", {}, { sanitize_options: { attributes: %w[target href] } })
       #   # => "<p><a target=\"_blank\" href=\"http://example.com\">Continue</a></p>"
       def simple_format(text, html_options = {}, options = {})
-        wrapper_tag = options.fetch(:wrapper_tag, :p)
+        wrapper_tag = options.fetch(:wrapper_tag, :p).presence || :p
```

Comment:
```suggestion
        wrapper_tag = options[:wrapper_tag] || "p"
```

Nitpick, but since we'll end up calling `wrapper_tag.to_s` down the line, we might as well directly use a string.

We can also get rid of `fetch`

Reply:
But even if are not raising an error for the empty string `""`, that again results in empty tags `<></>`.

Reply:
As I said before I'd rather validate the string, so that would go on another line anyway.

Reply:
Actually, the validation should probably be in `content_tag`

Reply:
That makes sense